testing/utilities.py

```diff
@@ -66,6 +67,30 @@ def run_tests(test_file_name):
     log_file = os.path.splitext(test_file_name)[0] + '.log'
     with open(os.path.join(get_root_path(),'testing',log_file), 'w') as f:
         json.dump(log_json, f)
+        
+def create_mimic_boptest(testcase_name):
+    '''Creates a testcase directory in root mimicking the boptest docker.
+    
+    '''
+    
+    # Get testcase directory
+    testcase_dir = os.path.join(get_root_path(),'testcases', testcase_name)
+    # Make models directory
+    os.mkdir(os.path.join(get_root_path(),'models'))
```

Comment:
If a previous test fails for whatever reason it won't be teared down and thus the `models` folder will remain in the directory. This causes a ripple effect to subsequent test that fail because of the error: `Cannot create a file when that file already exists`. I suggest to check before this line whether the folder `models` already exists and call `remove_mimic_boptest()` in case it does.

Reply:
Fixed in https://github.com/ibpsa/project1-boptest/pull/229/commits/fbc83b6bc4b6919523e318c2b36392a7389513fd.